src/components/App.jsx

```diff
@@ -38,6 +39,7 @@ class App extends Component {
         <main class={styles['fil-content']}>
           <Topbar />
           { children }
+          { toastMessage && <Toast />}
```

Comment:
Why not?
Containers and Components are both Components as known as Dumb Components and Smart Components.

What differentiate smart and dumb? Within `redux` and `react-redux`, a Smart Component is a component that fetches data from state or store's dispatch (*actions*), so you could have a Component tree with different layers including SC or DC in any order.

Do you agree @goldoraf?